### PR TITLE
Review logging level and configuration

### PR DESCRIPTION
**Pull request by bersace**
cf. #219 

cc @mhagander

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -103,3 +102,39 @@ sample or CLI help.
...
+``` console
+$ VERBOSITY=DEBUG ldap2pg
+[ldap2pg.config        INFO] Starting ldap2pg 4.9.
+[ldap2pg.config       DEBUG] Trying ./ldap2pg.yml.
+...zillions of debug messages
+[ldap2pg.psql         DEBUG] Closing Postgres connexion to 'postgres://postgres@postgres.ldap2pg.docker/postgres'.
+$ ldap2pg -v  # Same as above
+...
+$ ldap2pg -q  # no info, just changes, warnings and errors.
+Running in dry mode. Postgres will be untouched.
```
Comment (bersace) [inline docs/cli.md]: @mhagander, can you review the doc ? Is it clear for you and does what you expect ?

Especially, ldap2pg always warns about whether it's dry or real mode. Is it ok for you use case ?

Comment (bersace) [reply docs/cli.md]: I can move `CHANGE` level above `WARNING` so that `VERBOSITY=CHANGES` does not include warnings. Tell me.

**Comment by bersace** [comment]
@mhagander :up: ;-)

**Comment by mhagander** [comment]
Apologies for the delay on this -- github notices remain in a side-tag, and it took me some time to get around to it :)

I think this seems good. Personally I'd want it in "changes,warnings and errors" only, but there are definitely people who want to have just "warnings and errors", so I think it's good thing to support both those.

With one possible exception -- it should warn if it's in dry mode, but it should not warn if it's in real mode. If it does warn in real mode, then it will always warn _something_ which means there is no way to run it quietly in cron..

I haven't looked at the code to see if it works :), but the documented behavior definitely sounds good!

Thanks!

**Comment by bersace** [comment]
Let's go.

**Pull request merged**